sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeErrors.cs

```diff
@@ -59,5 +59,13 @@ public static DataLakeAclChangeFailedException ChangeAclFailed(Exception excepti
                 $"{nameof(DataLakeAclChangeFailedException.ContinuationToken)}={continuationToken} after addressing the error.",
                 exception,
                 continuationToken);
+
+        internal static void VerifyHttpsCustomerProvidedKey(Uri uri, DataLakeCustomerProvidedKey? customerProvidedKey)
```

Comment:
Do we normally do verification in the errors class? I feel like we usually just assemble them here when it has already been determined we need to throw.

Reply:
Good question, in this case I pattern matched [BlobErrors](https://github.com/Azure/azure-sdk-for-net/blob/9d2509baa0d972971518ad43483f3a35937a216a/sdk/storage/Azure.Storage.Blobs/src/BlobErrors.cs#L26).